pkg/api/types.go

```diff
@@ -515,8 +524,8 @@ type PodSpec struct {
 // PodStatus represents information about the status of a pod. Status may trail the actual
 // state of a system.
 type PodStatus struct {
-	Phase PodPhase `json:"phase,omitempty"`
-
+	Phase      PodPhase       `json:"phase,omitempty"`
+	Conditions []PodCondition `json:"Condition,omitempty"`
```

Comment:
As per @bgrant0607's suggestion, this is intended to be like [NodCondition](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L768-L812) as he expects to add more conditions in the future. I stopped short of creating the PodCondition struct but I could in order to fully mirror NodeCondition api. Until more PodConditions are added, Ready bool would be a functional substitute for Conditions. So I guess it comes down to whether we want to add PodCondition to the api now or hold off until later.